akka-actor-typed/src/main/scala/akka/actor/typed/WrappingBehavior.scala

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (C) 2009-2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.actor.typed
+
+// FIXME see if we can completely eliminate this instead
```

Comment:
yes, we might want to think twice before making it public. Might be enough with the interceptor for most things.